roscala/src/main/scala/coop/rchain/rosette/prim/Tuple.scala

```diff
@@ -0,0 +1,37 @@
+package coop.rchain.rosette.prim
+
+import coop.rchain.rosette.{
+  Ctxt,
+  Fixnum => RFixnum,
+  Ob,
+  RblBool,
+  Tuple => RTuple
+}
+import coop.rchain.rosette.macros.{checkArgumentMismatch, checkTypeMismatch}
```

Comment:
Seems like you don't need the `checkTypeMismatch` import.

Reply:
Leaving it because I'll need it later